File: examples/working-with-json/working-with-json.ts
const person = {
    name: "John Doe" 
  }
// stringify
const personJson = JSON.stringify(person);
console.log(personJson); // '{ "name": "John Doe", "age": 33 }'

// parse
const personObject = JSON.parse('{ "name": "John Doe", "age": 33 }');
console.log(personObject.name) // John Doe

File: examples/working-with-json/working-with-json.dart
import 'dart:convert';

Map person = {
  "name": "John Doe"
};

// stringify
var personJson = jsonEncode(person);
print(personJson); // '{ "name": "John Doe", "age": 33 }'

// parse
Map personMap = jsonDecode('{ "name": "John Doe", "age": 33 }');
print(personMap["name"]); // John Doe
